gettsim/transfers/rente.py

```diff
@@ -295,4 +423,498 @@ def ges_rente_regelaltersgrenze(geburtsjahr: int, ges_rente_params: dict) -> flo
             "intercepts_at_lower_thresholds"
         ],
     )
+
+    return out
+
+
+def ges_rente_frauen_altersgrenze(
+    geburtsjahr: int,
+    geburtsmonat: int,
+    ges_rente_params: dict,
+) -> float:
+    """Calculates the age, at which a women is eligible to claim the full pension
+    (without deductions). This pension scheme allows for early retirement
+    from age 60 with deductions. Hence this threshold is needed as reference
+    for calculating the zugangsfaktor.
+
+
+    Parameters
+    ----------
+    geburtsjahr
+        See basic input variable :ref:`geburtsjahr <geburtsjahr>`.
+    geburtsmonat
+        See basic input variable :ref:`geburtsmonat <geburtsmonat>`.
+    ges_rente_params
+        See params documentation :ref:`ges_rente_params <ges_rente_params>`.
+
+    Returns
+    -------
+    returns the lowest full retirement age for women.
+
+    """
+    # From 1945 on, the altersgrenze of women is equal to the Regelaltersgrenze which
+    # is indpendendent of the birth month and only depends on the birth year.
+    if geburtsjahr < 1945:
+        x = geburtsjahr + (geburtsmonat - 1) / 12
+    else:
+        x = geburtsjahr
+
+    out = piecewise_polynomial(
+        x=x,
+        thresholds=ges_rente_params["altersgrenze_für_frauen_abschlagsfrei"][
+            "thresholds"
+        ],
+        rates=ges_rente_params["altersgrenze_für_frauen_abschlagsfrei"]["rates"],
+        intercepts_at_lower_thresholds=ges_rente_params[
+            "altersgrenze_für_frauen_abschlagsfrei"
+        ]["intercepts_at_lower_thresholds"],
+    )
+
+    return out
+
+
+def _ges_rente_langjährig_altersgrenze(
+    geburtsjahr: int,
+    geburtsmonat: int,
+    ges_rente_params: dict,
+) -> float:
+    """Calculates the age, at which a long term insured person (at least 35 years ) is
+    eligible to claim the full pension (without deductions). This pension scheme allows
+    for early retirement from age 63 with deductions. Hence this threshold is needed as
```

Comment:
The 63 seems like a number that could change? IIRC, the 35 above is unlikely to ever change?

If so, we should avoid the hard-coded 63 or add something like "since 200x".

Reply:
The 35 is unlikely to ever change. According to patterns in the past, they will rather they will invent another term and make the pension system more complicated. 
There is no hard coded 63 in this function - I changed the docstring.